notebook/Makefile

```diff
@@ -1,32 +1,43 @@
-.PHONY: build push run-docker run deploy clean-workers build-worker push-worker
+.PHONY: build push run-docker run deploy
 
 PROJECT = $(shell gcloud config get-value project)
+BUILT = $(shell ls -1 images | awk '{print "build/images/" $$0 "/_BUILT"}')
+PUSHED = $(shell ls -1 images | awk '{print "build/images/" $$0 "/_PUSHED"}')
 
-build: push-worker
+build/images/%/_BUILT: images/%/Dockerfile $(shell find images/$*)
```

Comment:
You forgot to check in these docker files.

Reply:
oops